posthog/api/test/dashboards/test_dashboard.py

```diff
@@ -18,6 +19,9 @@
 from posthog.utils import generate_cache_key
 
 
+@override_settings(
+    PERSON_ON_EVENTS_OVERRIDE=False
+)  # :KLUDGE: avoid making a bunch of extra queries which would normally be cached
```

Comment:
does this prevent the persons on events test suite from running correctly for these tests?

Reply:
ugh, yes, I think it does. `settings.PERSON_ON_EVENTS_OVERRIDE` is checked before `PERSON_ON_EVENTS_ENABLED` env that's enabled in the `.github` folder. I'll think about it, but this is obviously a no go as is.